route/example_measure_test.go

```diff
@@ -0,0 +1,94 @@
+package route_test
+
+import (
+	"fmt"
+
+	"github.com/nextmv-io/sdk/route"
+)
+
+func ExampleConstant() {
+	constant := route.Constant(1.234)
+
+	fmt.Println(constant.Cost(1, 0))
+	fmt.Println(constant.Cost(0, 0))
+	fmt.Println(constant.Cost(0, 1))
+
+	// Output:
+	// 1.234
+	// 1.234
+	// 1.234
+}
+
+func ExampleOverride() {
+	defaultByIndex := route.Constant(1.234)
+
+	overrideByIndex := route.Constant(4.321)
+
+	condition := func(from, to int) bool {
+		return from > 0
+	}
+
+	override := route.Override(
+		defaultByIndex,
+		overrideByIndex,
+		condition,
+	)
+
+	fmt.Println(override.Cost(0, 0))
+	fmt.Println(override.Cost(0, 1))
+	fmt.Println(override.Cost(1, 0))
+
+	// Output:
+	// 1.234
+	// 1.234
+	// 4.321
+}
+
+func ExampleHaversineByPoint() {
+	byPoint := route.HaversineByPoint()
+
+	measure := byPoint.Cost(route.Point{1, 2}, route.Point{4, 5})
```

Comment:
Can we make these coordinates something more real? That way the output is not extremely large and people understand these are points on the Earth, not indices.

Reply:
@davidrijsman This is still not acknowledged.